// Copyright 2019 Zeppelin Bend Pty Ltd
// This file is part of cimbend.
// 
// cimbend is free software: you can redistribute it and/or modify
// it under the terms of the GNU Affero General Public License as published by
// the Free Software Foundation, either version 3 of the License, or
// (at your option) any later version.
// 
// cimbend is distributed in the hope that it will be useful,
// but WITHOUT ANY WARRANTY; without even the implied warranty of
// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
// GNU Affero General Public License for more details.
// 
// You should have received a copy of the GNU Affero General Public License
// along with cimbend.  If not, see <https://www.gnu.org/licenses/>.


syntax = "proto3";
option java_multiple_files = true;
option java_package = "com.zepben.protobuf.cim.iec61968.assetinfo";
package zepben.cim.iec61968.assetinfo;

import "zepben/cim/iec61968/assetinfo/CableInfo.proto";
import "zepben/cim/iec61968/assetinfo/OverheadWireInfo.proto";
import "zepben/cim/iec61968/assetinfo/TransformerEndInfo.proto";

/**
 * Cable data.
 */
message AssetInfo {

    oneof assetInfo {
        iec61968.assetinfo.CableInfo cableInfo = 1;
        iec61968.assetinfo.OverheadWireInfo overheadWireInfo = 2;
        iec61968.assetinfo.TransformerEndInfo transformerEndInfo = 3;
    }

}
